docs/modules/ROOT/pages/flow.adoc

```diff
@@ -0,0 +1,72 @@
+= Deploy Flow
+
+:description: In this tutorial, you'll deploy a Flow application using Hazelcast Platform Operator for Kubernetes.
+
+{description}
+
+== Prerequisites
+
+- Flow requires the Management Center application to run and be configured properly. The Management Center application should be configured to use FLow.
+
```

Comment:
Please mention FLOW MC image should be used here --> `docker.io/hazelcast/management-center-flow`

Reply:
I have added it to the example config with the following description